okhttp/src/main/java/okhttp3/internal/Platform.java

```diff
@@ -176,6 +193,13 @@ private static Platform findPlatform() {
     return new Platform();
   }
 
+  /**
+   * Is Java 9.  Does not include Java 10 or later.
```

Comment:
1 tangential comment, OptionalMethod is broken in JDK9 because it uses the concrete type which can be offlimits instead of the public supertype that declares the method.